tools/lint

```diff
@@ -15,6 +15,7 @@ EXCLUDED_FILES = [
 def run() -> None:
     parser = argparse.ArgumentParser()
     add_default_linter_arguments(parser)
+    parser.add_argument('--no-gitlint', action='store_true', help='Disable gitlint')
```

Comment:
zulint already has an option for this: #659.

Reply:
Ok, I will create a PR soon, fixing the required changes.